tests/handlers/filesystem/test_jffs2.py

```diff
@@ -57,6 +59,10 @@ def get_valid_jffs2_old_be_header():
     )
 
 
+def calculate_crc(header: Instance):
```

Comment:
I would make this the other way around: put this into the jffs2 handler, and call it from this test instead.